### DISCUSSION TITLE
📢 鸢种交流区，启动！

### DISCUSSION THREAD

**Discussion by syoius** [syoius/MaaYuan#42] [Announcements]
## 👋 欢迎！

这里是 MaaYuan 使用交流区。在这里，您可以：

* 提出您想了解的问题
* 分享想法
* 与其他社区成员互动交流

希望您能以开放的心态欢迎他人。请记住，这是我们共同建设的社区 💪

**Comment by afjp00** [comment]
你好，我定了张昭是首位展示的密探，才第3级，不到突破的阶段，但是软件无法识别，一直卡在升级那

**Reply by syoius** [reply]
看起来是OCR错误的样子🤔

可以打包传一下maa目录下的debug文件夹过来（最好把之前的删了、单独运行一下日常收尾，关闭maa后再打包上传），我看下认成什么了
